src/nodes/corenodes/draw/add_text_node.py

```diff
@@ -0,0 +1,99 @@
+# ----------------------------------------------------------------------------
+# Gimel Studio Copyright 2019-2021 by Noah Rahm and contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------
+
+try:
+    import OpenImageIO as oiio
+except ImportError:
+    print("""OpenImageIO is required! Get the python wheel for Windows at:
+     https://www.lfd.uci.edu/~gohlke/pythonlibs/#openimageio""")
+
+from gimelstudio import api
+
+
+class AddTextNode(api.Node):
+    def __init__(self, nodegraph, _id):
+        api.Node.__init__(self, nodegraph, _id)
+
+    @property
+    def NodeMeta(self):
+        meta_info = {
+            "label": "Add Text",
+            "author": "Gimel Studio",
+            "version": (0, 5, 0),
+            "category": "DRAW",
+            "description": "Adds a text overlay to an image.",
+        }
+        return meta_info
+
+    def NodeInitProps(self):
+        self.font = api.ChoiceProp(
+            idname="font",
+            default="Arial",
+            choices=["Arial", "Calibri"],
```

Comment:
Will need to work on a cross-platform way of getting all of the fonts for each OS